configureWorkspace/configure.ts

```diff
@@ -153,15 +153,17 @@ LABEL Name=${serviceName} Version=${version}
 EXPOSE ${port}
 
 WORKDIR /app
-ADD . /app
+COPY . .
+
+# install dependencies if dependencies file is present
+RUN python3 -m pip install -r requirements.txt || echo ‘No requirements.txt - skipping pip install’
 
 # Using pip:
-RUN python3 -m pip install -r requirements.txt
 CMD ["python3", "-m", "${serviceName}"]
 
 # Using pipenv:
-#RUN python3 -m pip install pipenv
-#RUN pipenv install --ignore-pipfile
+#RUN python3 -m pip install pipenv  || echo ‘No requirements.txt - skipping pip install’
```

Comment:
This failing has nothing to do with a missing `requirements.txt` file and everything to do with `pip` failing to install `pipenv` which would be fatal for anyone choosing to use `pipenv`, so this change should be reverted.